app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsAddDescriptionsItemFragment.kt

```diff
@@ -140,7 +139,7 @@ class SuggestedEditsAddDescriptionsItemFragment : Fragment() {
         }
 
         viewArticleExtract.text = StringUtil.fromHtml(sourceSummary!!.extractHtml)
-        if (TextUtils.isEmpty(sourceSummary!!.thumbnailUrl)) {
+        if (sourceSummary!!.thumbnailUrl.isNullOrEmpty()) {
```

Comment:
For the thumbnails URLs, I think it would be better if we use `isNullOrBlank()` here.